### PR TITLE
small random notes

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

File: experimental/add_to_app/README.md [hunk 1]
```diff
@@ -16,7 +16,22 @@ a standalone Flutter module called `example_module`.
...
+a `FlutterActivity` containing widgets from the module. This project brings
+in the Flutter module's Android host wrapper as a gradle sub-module. To set
+it up prior to running it for the first time:
+
+```bash
+cd example_module
+flutter packages get
+```
+
+Then open the SimpleAndroidApp folder in Android Studio. Gradle with sync the
```
Comment (RedBrogdon) [inline experimental/add_to_app/README.md]: I'm not able to parse "Gradle with sync the Flutter module in as a sub-module." Is there a transposition in there, or am I just not reading it right?

File: experimental/add_to_app/example_module/pubspec.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: example_module
-description: A simple module used to demo Flutter's Add2App features
+description: A simple module used to demo Flutter's Add-to-App features
```
Comment (RedBrogdon) [inline experimental/add_to_app/example_module/pubspec.yaml]: I could have sworn I grepped the code and fixed all of these. Thanks for catching it.

File: experimental/add_to_app/README.md [hunk 3]
```diff
@@ -26,10 +41,15 @@ uses CocoaPods, so prior to running it for the first time, use this command
 to set up the workspace file:
 
 ```bash
-cd SimpleIOSExample
+cd example_module
```
Comment (RedBrogdon) [inline experimental/add_to_app/README.md]: nit: Consider adding the step of cd-ing into the module and getting packages to the module section above (e.g. "Before running either of the projects below, you'll need to get packages for the Flutter module. To do that...")

File: experimental/add_to_app/README.md (hunk 1 above)
Comment (xster) [reply experimental/add_to_app/README.md]: ah brain melt. Revised

File: experimental/add_to_app/README.md (hunk 3 above)
Comment (xster) [reply experimental/add_to_app/README.md]: oh ya, good idea

**Pull request merged**